preferences/components/pci-hole-64/kustomization.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: kustomize.config.k8s.io/v1alpha1
+kind: Component
+
+patches:
+  - path: ./pci-hole-64.yaml
```

Comment:
It's called `pcihole64` everywhere else but that is super nit-picky me

Reply:
Adjusted